croptalk/prompts_llm.py

```diff
@@ -28,7 +28,8 @@
 
 REPHRASE_TEMPLATE = """\
 Given the following conversation and a follow up question, identify whether the question is a follow-up question.\
-If it is, rephrase the question to be a standalone question. If it is not, return the question as is.\
+If it is, rephrase the question to be a standalone question. If it is not, return the question as is and nothing else.\
```

Comment:
So this is the tentative fix for the "fix confidence chain" issue?

Reply:
Yes, do you want me to include it in another PR?